packages/stripe_checkout/lib/src/platforms/stripe_checkout_mobile.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/widgets.dart';
+import 'package:stripe_checkout/src/platforms/checkout.dart';
+
+Future<CheckoutResponse?> redirectToCheckout({
```

Comment:
apparently the old method does not work anymore 🤷‍♂️  not sure if this is correct though